protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -355,14 +357,15 @@ func (k Keeper) CanUpdateSubaccounts(
 		time.Now(),
 		metrics.CanUpdateSubaccounts,
 		metrics.Latency,
+		// TODO: Add UpdateType as label
```

Comment:
Ideally can add this as a label to the existing metrics.
- [ ] Add `UpdateType` as a label within this PR / create ticket to do so later.